api/spec/integration/api/v2/platform/states_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'swagger_helper'
+
+describe 'States API', swagger: true do
+  include_context 'Platform API v2'
+
+  path '/api/v2/platform/states' do
+    get 'Returns a list of States' do
+      tags 'States'
+      security [ bearer_auth: [] ]
+      operationId 'states-list'
+      description 'Returns a list of States'
+      before { create_list(:state, 2) }
+
+      it_behaves_like 'records returned'
+      it_behaves_like 'authentication failed'
+    end
+  end
+
+  path '/api/v2/platform/states/{id}' do
+    let(:id) { create(:state).id }
+
+    get 'Returns a State' do
+      tags 'States'
+      security [ bearer_auth: [] ]
+      operationId 'show-state'
```

Comment:
here we have show-state, and list if states-list - it's inconsistent

Reply:
I do not follow, we have that for other resources: addresses-list & show-address etc.
In the shared examples: operationId "#{resource_name.parameterize.pluralize.to_sym}-list" & "show-#{resource_name.parameterize.to_sym}"